src/dlls/mscoree/unixinterface.cpp

```diff
@@ -178,13 +178,13 @@ static void ExtractStartupFlagsAndConvertToUnicode(
 //
 extern "C"
 int coreclr_initialize(
-            const char* exePath,
```

Comment:
Regarding the parameters, it really depends on where you look in the codebase. Some code uses what you said, some aligns all parameters with the opening brace, some uses what I've used.
It would be nice if you could revert it here.